programs/disks/Path.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CLICKHOUSE_PATH_H
+#define CLICKHOUSE_PATH_H
+
+#pragma once
+#include <string>
+
+namespace DB
+{
+
+struct Path
+{
+public:
+    Path();
+
+    void parsPath(std::string toPars);
+    void setDiskName(std::string name);
```

Comment:
If you pass default disk name(or overriden via args) this method is useless
And probably you can pass path to parse in constructor too